dicom-archive/profileTemplate.pl

```diff
@@ -88,6 +91,7 @@ sub getSubjectIDs {
         #     ? 1 : 2;
         # When createVisitLabel is set to 0, $subjectID{'SubprojectID'} is ignored.
         
+        $subjectID{'ProjectID'} = 1;
```

Comment:
I am not sure I would give a default value to the `ProjectID`. I think it would be better to leave it undefined and have the MRI pipeline complain that if was not set instead of setting the project ID of the newly created candidate to 1 (which might not be the correct value). @cmadjar Can you weigh in on this?

Reply:
I agree with @nicolasbrossard. Since `ProjectID` can be `NULL` I would leave the default to be `undef`.

Reply:
I get confused by [this change](https://github.com/aces/Loris/blob/973fb261e420ae23696ddda628bfe2831260353e/SQL/Archive/24.0/2019-07-01_fix_project_in_session.sql).
Since v24.0 session ProjectID cannot be null. createVisitLabel is set to 1 by default, so setting Project to undef will trigger an INSERT error when attempting to create a session. Can we fix this by setting createVisitLabel to 0 by default?